libs/openchallenges/ui/src/lib/search-dropdown-filter/search-dropdown-filter.component.scss

```diff
@@ -54,10 +58,43 @@
   }
 }
 
+.loader-container {
```

Comment:
Do these styles defined the placeholders displayed during loading? They look great in the preview you shared in the PR description.

Reply:
yes, PrimeNg has default placeholder for loaders called [skeleton](https://primeng.org/skeleton). We just need to figure out how to properly layout the loaders.